app/views/course/announcements/_announcement.html.slim

```diff
@@ -0,0 +1,20 @@
+div class=(announcement.currently_valid? ? '' : 'future')
```

Comment:
might not want to use future, since it will also apply to expired announcements.